packages/templates/src/cli.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-await-in-loop, no-console */
+
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import puppeteer from 'puppeteer';
+
+const screenshotsPath = `build/template-posters/`;
+
+fs.mkdirSync(screenshotsPath, { recursive: true });
+
+(async () => {
+  const browser = await puppeteer.launch({
+    defaultViewport: null,
+    headless: true,
+    args: [`--window-size=1600,800`],
+  });
+  const page = await browser.newPage();
+  await page.setViewport({
+    width: 1600,
+    height: 800,
+    deviceScaleFactor: 2,
+  });
+  await page.goto(
+    'http://localhost:8899/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery'
+  );
+  await page.type('#user_login', 'admin');
+  await page.type('#user_pass', 'password');
+  await page.click('#wp-submit');
+  await page.waitForNavigation({ waitUntil: 'networkidle2' });
+
+  await page.waitForSelector('aria/Explore Templates');
+  await page.click('aria/Explore Templates');
+
+  // wait for templates to be rendered
+  await page.waitForSelector('div.templateGridItem');
+
+  // total templates to generate screenshots from
+  const templateCount = await page.evaluate(() => {
+    return document.querySelectorAll('.templateGridItem').length;
+  });
+
+  // Now let's go through each template
+  for (
+    let currentTemplate = 1;
+    currentTemplate <= templateCount;
+    currentTemplate++
+  ) {
+    // Get the template name and remove spaces to use as part of file name.
+    const templateName = await page.evaluate((gridItemId) => {
+      return document
+        .querySelector(
+          `div#template-grid-item-${gridItemId} div[data-testid="card-action-container"]`
+        )
+        .getAttribute('aria-label')
+        .replace(/\s/g, '');
+    }, currentTemplate);
```

Comment:
We should be getting the template slug from some `data-testid` instead so they 100% match the folder structure and the list in `getTemplates()`.

Reply:
Each template's `index.js` file could have a field `slug: "...",` that we than put into a `data-testid`

Reply:
you're right, thank you. i'll fix that.